setup.py

```diff
@@ -33,14 +33,14 @@
 
 tests_require = [
     'Flask-BabelEx>=0.9.3',
-    'Flask-Menu>=0.4.0',
+    'Flask-Menu>=0.5.0',
     'SQLAlchemy-Continuum>=1.2.1',
     'check-manifest>=0.25',
     'coverage>=4.0',
     'invenio-access>=1.0.0a11',
-    'invenio-accounts>=1.0.0b3',
-    'invenio-admin>=1.0.0b1',
-    'invenio-celery>=1.0.0b2',
+    'invenio-accounts>=1.0.0b9',
```

Comment:
Bump to latest released versions.

Reply:
I can do that, but then I might have a lot more to refactor in videos, to depend on new version. Quick discussion here too.